components/_patterns/01-atoms/02-text/00-headings/heading-1/heading-1.twig

```diff
@@ -1,17 +1,19 @@
 {#
 /**
  * Available variables:
+ * - h1_base_class - the base class name
  * - heading_1_modifiers - array of modifiers to add to the base classname
  * - heading_1_blockname - blockname prepended to the base classname
  * - heading_1_url - (optional) the url the heading should poing to
  * - heading_1 - the content of the heading (typically text)
  */
 #}
-<h1 class="heading-1{% for modifier in heading_1_modifiers %} heading-1--{{ modifier }}{% endfor %}{% if heading_1_blockname %} {{ heading_1_blockname }}__heading-1{% endif %}">
+<h1 class="{{ h1_base_class|default('heading-1') }}{% for modifier in heading_1_modifiers %} {{ h1_base_class|default('heading-1') }}--{{ modifier }}{% endfor %}{% if heading_1_blockname %} {{ heading_1_blockname }}__{{ h1_base_class|default('heading-1') }}{% endif %}">
   {% if heading_1_url %}
     {% include "@atoms/01-links/link/link.twig"
       with {
-        "link_blockname": "heading-1",
+        "link_base_class": heading_1_link_base_class,
+        "link_blockname": heading_1_link_blockname|default('heading-1'),
```

Comment:
Same question as the above line.  This is not an issue currently because we don't use `heading-1__link` for any styling but conceptually if we did have some styling here it would get overridden by changing the blockname.  Also does `heading-1__link` look a little strange to you or is this pretty normal to have both dashes and underscores?